.github/workflows/continuous_integration.yml

```diff
@@ -44,7 +44,7 @@ jobs:
 
         - name: Linux GCC 12
           os: ubuntu-22.04
-          extra_options: -DCMAKE_C_COMPILER=gcc-12 -DCMAKE_CXX_COMPILER=g++-12
+          extra_options: -DCMAKE_C_COMPILER=gcc-12 -DCMAKE_CXX_COMPILER=g++-12 -DCMAKE_DISABLE_PRECOMPILE_HEADERS=ON
```

Comment:
If you are trying to fix the continuous integration with the changes then a new test needs to be made for the newer compiler, not modifying one of the older ones.

Reply:
Done. You'll understand what I was trying to do when you replace GCC-12 with GCC-13, in a couple of months.

Reply:
Understood - we'll fight that battle when it comes to us. I'm hoping there's a better way to handle this, than disabling precompiled headers. Alexey usually manages the CI's and if he has to make changes to make the GCC-13 one work - he usually has another solution for those kind of issues.